### ISSUE TITLE
Compiling py34 on VS2013 x64

### ISSUE THREAD

**Issue by uclatommy** [mrwonko/PySixense#2]
I'm getting linking errors in VS2013.

> error LNK2038: mismatch detected for '_MSC_VER': value '1600' doesn't match value '1800' in dllmain.obj

Looks like sixense dlls are compiled on VS2010(1600) and it doesn't seem like they'll release dlls compiled with VS2013(1800).

Forked the py34 and attempted to bind to dll through dllimport (11f21c8a75f80cc9d8b0ffdd6d3d71c80387f713) without success. Is there any way to get this to compile on VS2013? Or is there somewhere where you store the precompiled binaries?

**Comment by mrwonko** [comment]
This shouldn't be a problem when using DLLs... Are you sure you're not linking to the static sixense libraries? Their name ends in `_s`.

Anyway, I never quite took the time to supply binaries, should've done that a long time ago... [done now](http://mrwonko.de/downloads/2015/pysixense-34.html). I've compiled using the Visual Studio 2015 Preview, which didn't complain at all regarding the version, but neither did 2013 back when I used that.
